dashboard/db/schema.rb

```diff
@@ -332,7 +332,7 @@
     t.index ["level_group_level_id"], name: "index_contained_levels_on_level_group_level_id", using: :btree
   end
 
-  create_table "course_offerings", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci" do |t|
+  create_table "course_offerings", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci" do |t|
```

Comment:
this looks like an unrelated change - also to the `course_versions` and `foorm_misc_surveys` tables

Reply:
@clareconstantine whats the best way to fix this?

Reply:
running `rails db:migrate` should regenerate schema.rb - if that doesn't fix it, then I think it means the local db is has changes in it that aren't represented by migrations, and we could probably write a migration locally to fix that, or @indhu-gunda could drop their db and recreate it. But hopefully rerunning the existing migrations is all that's needed!